src/aura/DragNDropCmp/DragNDropCmpController.js

```diff
@@ -0,0 +1,49 @@
+({  
+    // Load current profile picture
+    onInit: function(component) {
+        var action = component.get("c.getProfilePicture"); 
+        action.setParams({
+            parentId: component.get("v.recordId"),
+        });
+        action.setCallback(this, function(a) {
+            var image = a.getReturnValue();
```

Comment:
Before obtaining the response.getReturnValue(), we need to check if the state is "SUCCESS".